flair/trainers/trainer.py

```diff
@@ -75,7 +75,7 @@ def train(
         train_with_dev: bool = False,
         monitor_train: bool = False,
         monitor_test: bool = False,
-        embeddings_storage_mode: str = "cpu",
+        embeddings_storage_mode: str = "none",
```

Comment:
During training, it does make sense to have 'cpu' be the default option since many embeddings like FlairEmbeddings and the various transformer embeddings are expensive to produce. If the default is 'none' this will slow down training since embeddings have to be generated over and over again at each epoch.

Reply:
ok, I have added questions in my last issue :-)
Will restore the default option